NEWS.md

```diff
@@ -91,24 +91,24 @@ Note that each entry is kept to a minimum, see links for details.
     variable or method name even if it's a pseudo variable name such as
     `self`.
 
-*   non main-Ractors can get instance variables (ivars) of classes/modules
+*   Non main-Ractors can get instance variables (ivars) of classes/modules
     if ivars refer to shareable objects.
     [[Feature #17592]]
 
-*   A command syntax is allowed in endless method definition, i.e.,
+*   A command syntax is allowed in endless method definitions, i.e.,
     you can now write `def foo = puts "Hello"`.
     Note that `private def foo = puts "Hello"` does not parse.
     [[Feature #17398]]
 
 ## Command line options
 
 * `--disable-gems` is now explicitly declared as "just for debugging".
-  Never use it in any real-world code base.
+  Never use it in any real-world codebase.
   [[Feature #17684]]
 
 ## Core classes updates
 
-Outstanding ones only.
+Note: We're only listing outstanding class updates.
```

Comment:
I'm not sure if `Outstand ones only` on the previous sentence meant `Oustanding class updates` or `updates on outstanding `